help/en/releasenotes/current-draft-note.shtml

```diff
@@ -377,7 +377,12 @@
     <h3>Scripting</h3>
         <a id="Scripting" name="Scripting"/>
         <ul>
-            <li></li>
+            <li>Geoff Bunzxa provided sample scripts supporting bulk 
```

Comment:
Again, I _think_ this is a typo - shouldn't it be `Geoff Bunza`?

Reply:
Thanks! Have fixed.